### DISCUSSION TITLE
Shubham Paliwal : Resume Bundle X ATS Scoring

### DISCUSSION THREAD

**Discussion by ShubhamPaliwal03** [recodehive/Stackoverflow-Analysis#358] [Q&A] [answered]
Greetings, Sir

I am a final year (2025 passout) B.Tech Computer Science and Engineering student at Bikaner Technical University, Bikaner. I am into full stack web development (as of now PHP backend), and I have a keen interest in problem solving (DSA), I am currently learning MERN stack web development and Java Backend, and brushing up my DSA skills.

I am trying to apply for software development and web development intern posts, but unable to hear back from them, and my profile is not getting selected, I will be applying for Software Development Engineer, Java Developer and Web Developer roles this year, I would be grateful to you if you provide me a resume review, along with an ATS score, analysis, tips, and a ATS optimised version of my resume.

My Resume : [resume-shubham-paliwal-latest.pdf](https://github.com/user-attachments/files/16366255/resume-shubham-paliwal-latest.pdf)

Link to my Linkedin profile : https://www.linkedin.com/in/shubhammpaliwal/

Link to my Github profile : https://github.com/ShubhamPaliwal03

Thanks and Regards

**Comment by sanjay-kv** [comment]
[ATS Version of CV (19).docx](https://github.com/user-attachments/files/16388238/ATS.Version.of.CV.19.docx)
[subhma.pdf](https://github.com/user-attachments/files/16388242/subhma.pdf)

**Comment by vijaychandra1910** [comment, marked as answer]
Your resume is really great
